### PR TITLE
Fixed latent issues with worksheets CircleCI

### PR DESCRIPTION
**Pull request by andreweduffy**
Was not generating website-config or database tables before. Now the (skeleton) tests should pass.

### CODE REVIEW

**Comment by andreweduffy** [comment]
@percyliang or @kashizui This should be a quick check

File: circle.yml [hunk 1]
```diff
@@ -3,9 +3,11 @@ machine:
     version: 2.7.5
 
 dependencies:
-  pre:
-    - pip install -r codalab/requirements/common.txt
+  override:
+    - mkdir ~/.codalab
```
Comment (percyliang) [inline circle.yml]: This should be created automatically - I think there's no need for explicit creation.

**Comment by percyliang** [comment]
Otherwise, LGTM

**Comment by andreweduffy** [comment]
I added that because it didn't seem to be creating ~/.codalab, where is that created automatically? Usually that's created by calling cl for the first time I thought





On Sun, Jan 24, 2016 at 5:55 PM -0800, "Percy Liang" <notifications@github.com> wrote:












Otherwise, LGTM

**Comment by percyliang** [comment]
It is called when you call 'cl', which of course isn't being called here.
But you can mkdir when you create website-config.json.

**Comment by andreweduffy** [comment]
Ah okay, so you're saying just move it down to the block below?

**Comment by percyliang** [comment]
Yeah, it's a minor change, but makes it more obvious why the mkdir is
necessary.

**Pull request merged**